github/GithubObject.py

```diff
@@ -285,44 +347,57 @@ def format_params(params):
             params=", ".join(list(format_params(params))),
         )
 
+    def _initAttributes(self):
+        raise NotImplementedError("BUG: Not Implemented _initAttributes")
+
+    def _useAttributes(self, attributes):
+        raise NotImplementedError("BUG: Not Implemented _useAttributes")
 
-class NonCompletableGithubObject(GithubObject):
     def _completeIfNeeded(self):
+        raise NotImplementedError("BUG: Not Implemented _completeIfNeeded")
+
+
+class NonCompletableGithubObject(GithubObject):
+    def _completeIfNeeded(self) -> None:
         pass
 
 
 class CompletableGithubObject(GithubObject):
-    def __init__(self, requester, headers, attributes, completed):
+    def __init__(
+        self,
+        requester: "Requester",
+        headers: Dict[str, Union[str, int]],
+        attributes: Dict[str, Any],
+        completed: bool,
+    ):
         super().__init__(requester, headers, attributes, completed)
         self.__completed = completed
 
-    def __eq__(self, other):
+    def __eq__(self, other: Any) -> bool:
         return other.__class__ is self.__class__ and other._url.value == self._url.value
 
     def __hash__(self):
         return hash(self._url.value)
 
-    def __ne__(self, other):
+    def __ne__(self, other: Any) -> bool:
         return not self == other
 
-    def _completeIfNotSet(self, value):
-        if value is NotSet:
+    def _completeIfNotSet(self, value: Attribute) -> None:
+        if isinstance(value, _NotSetType):
             self._completeIfNeeded()
 
-    def _completeIfNeeded(self):
+    def _completeIfNeeded(self) -> None:
```

Comment:
Indentation ?

Reply:
what?

Reply:
Never mind, it looked wrong on my phone, but it's fine.